tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -130,28 +129,27 @@ def evaluate():
 # ------------------------------------------------------------------------------
 # ------------------------------------------------------------------------------
 
-if not FLAGS.dryrun:
-  ds_train, info = ModelNet40.load(split="train", with_info=True)
-  num_examples = info.splits["train"].num_examples
-  ds_train = ds_train.shuffle(num_examples, reshuffle_each_iteration=True)
-  ds_train = ds_train.repeat(FLAGS.num_epochs)
-  ds_train = ds_train.batch(FLAGS.batch_size)
-  ds_test = ModelNet40.load(split="test").batch(FLAGS.batch_size)
+ds_train, info = ModelNet40.load(split="train", with_info=True)
```

Comment:
note ds_train is the standard name used by TFDS in many examples.
I saw you changed it, then reverted it back now.

Reply:
changed it to avoid pylint warnings about overriding globals :)